extensions/extensions-docker/aissemble-jenkins/aissemble-jenkins-controller/src/main/resources/docker/Dockerfile

```diff
@@ -5,11 +5,11 @@ LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
 USER root
 USER jenkins
 
```

Comment:
I: The changes in versions represent IA issues flagged during the docker build of this container.